lbrynet/lbryfilemanager/EncryptedFileDownloader.py

```diff
@@ -63,8 +68,41 @@ def _save_claim(name, txid):
             d.addCallbacks(_save_claim_id, lambda err: _notify_bad_claim(name, txid))
             return d
 
+        def _check_file_availability():
```

Comment:
1. please, please avoid nested / inner functions. They're hard to read, can create huge messes with scoping and are impossible to test in isolation.
 2. Instead of adding these methods to this class / function (making it bigger and more complicated) - I'd suggest creating a new class that only does what you want and then use composition to combine the existing functionality with the new stuff.